luda/editor/client/src/timeline/DefaultTrackBody.ts

```diff
@@ -1,17 +1,34 @@
-import { ColorUtil, MouseButton, Rect, Render, BorderPosition } from "namui";
+import {
+  ColorUtil,
+  MouseButton,
+  Rect,
+  Render,
+  BorderPosition,
+  Mathu,
+} from "namui";
 import { Track, TimelineState } from "./type";
-import { ClipComponent } from "./clip/ClipComponent";
-import { Sash } from "./clip/Sash";
+import { Clip } from "../type";
+import { renderClip } from "./clip/renderClip";
+import { isSubtitleClip } from "../clipTypeGuard";
 
 export const DefaultTrackBody: Render<
-  TimelineState,
+  { timelineState: TimelineState; track: Track },
   {
     width: number;
     height: number;
-    track: Track;
   }
 > = (state, props) => {
-  const { clips } = props.track;
+  const { clips } = state.track;
+  let selectedClip: Clip | undefined = undefined;
```

Comment:
I think this is unused variable. let's remove it

Reply:
Yes. That's unused. Removed
with https://github.com/NamseEnt/namseent/pull/18/commits/7f1324abc6a5ad3addf6ded835c956a5d553411e